runtime/rococo/src/bridge_messages.rs

```diff
@@ -0,0 +1,409 @@
+	pub type FromWococoMessagePayload = messages_target::FromBridgedChainMessagePayload<
+		AtRococoWithWococoMessageBridge
+	>;
+
+	/// Encoded Rococo Call as it comes from Wococo.
+	pub type FromWococoEncodedCall = messages_target::FromBridgedChainEncodedMessageCall<crate::Call>;
+
+	/// Call-dispatch based message dispatch for Wococo -> Rococo messages.
+	pub type FromWococoMessageDispatch = messages_target::FromBridgedChainMessageDispatch<
+		AtRococoWithWococoMessageBridge,
+		crate::Runtime,
+		pallet_balances::Pallet<crate::Runtime>,
+		crate::AtRococoFromWococoMessagesDispatch,
+	>;
+}
+
+/// This module contains definitions that are used by the messages pallet instance, 'deployed' at Wococo.
+mod at_wococo {
+	use super::*;
+
+	/// Message bridge that is 'deployed' at Wococo chain and connecting it to Rococo chain.
+	#[derive(RuntimeDebug, Clone, Copy)]
+	pub struct AtWococoWithRococoMessageBridge;
+
+	impl MessageBridge for AtWococoWithRococoMessageBridge {
+		const THIS_CHAIN_ID: ChainId = WOCOCO_CHAIN_ID;
+		const BRIDGED_CHAIN_ID: ChainId = ROCOCO_CHAIN_ID;
+		const RELAYER_FEE_PERCENT: u32 = 10;
+
+		type ThisChain = WococoAtWococo;
+		type BridgedChain = RococoAtWococo;
+		type BridgedMessagesInstance = crate::AtRococoWithWococoMessagesInstance;
+
+		fn bridged_balance_to_this_balance(bridged_balance: bp_rococo::Balance) -> bp_wococo::Balance {
+			bridged_balance
+		}
+	}
+
+	/// Message payload for Wococo -> Rococo messages as it is seen at the Wococo.
+	pub type ToRococoMessagePayload = messages_source::FromThisChainMessagePayload<AtWococoWithRococoMessageBridge>;
+
+	/// Message verifier for Wococo -> Rococo messages at Wococo.
+	pub type ToRococoMessageVerifier = messages_source::FromThisChainMessageVerifier<AtWococoWithRococoMessageBridge>;
+
+	/// Message payload for Rococo -> Wococo messages as it is seen at Wococo.
+	pub type FromRococoMessagePayload = messages_target::FromBridgedChainMessagePayload<
+		AtWococoWithRococoMessageBridge,
+	>;
+
+	/// Encoded Wococo Call as it comes from Rococo.
+	pub type FromRococoEncodedCall = messages_target::FromBridgedChainEncodedMessageCall<crate::Call>;
+
+	/// Call-dispatch based message dispatch for Rococo -> Wococo messages.
+	pub type FromRococoMessageDispatch = messages_target::FromBridgedChainMessageDispatch<
+		AtWococoWithRococoMessageBridge,
+		crate::Runtime,
+		pallet_balances::Pallet<crate::Runtime>,
+		crate::AtWococoFromRococoMessagesDispatch,
+	>;
+}
+
+#[cfg(test)]
+mod tests {
+	use bridge_runtime_common::messages;
+	use parity_scale_codec::Encode;
+	use super::*;
+
+	#[test]
+	fn ensure_rococo_messages_weights_are_correct() {
+		// **NOTE**: the main purpose of this test is to be sure that any message that is sumbitted
+		// to (any) inbound lane in Rococo<>Wococo bridge can be delivered to the bridged chain.
+		// Since we deal with testnets here, in case of failure + urgency:
+		//
+		// 1) ping bridges team about this failure;
+		// 2) comment/remove the test.
```

Comment:
```suggestion
		// 2) comment out or ignore the test.
```

Reply:
You can't simply ignore failing test, if it blocks CI

Reply:
(unless you mean `#[ignore]`, of course)

Reply:
Yeah I meant `#[ignore]`. Should we make that more explicit in the comment?